README.md

```diff
@@ -1,4 +1,6 @@
-# [![Build Status](https://travis-ci.org/aces/Loris.svg?branch=master)](https://travis-ci.org/aces/Loris) [![Documentation Status](https://readthedocs.org/projects/acesloris/badge/?version=latest)](https://acesloris.readthedocs.io/en/latest/?badge=latest)
+<img src="./loris-logo.svg" width="50%">
```

Comment:
35% looks better to me in preview mode - not too tiny, you're right. 
```suggestion
<img src="./loris-logo.svg" width="35%">
```

Reply:
The biggest problem with Readmes is that people don't read them, is what i'm really thinking :)